README.md

```diff
@@ -71,7 +74,8 @@ def handle_data(found_data):
 # List of macs of sensors which will execute callback function
 macs = ['AA:2C:6A:1E:59:3D', 'CC:2C:6A:1E:59:3D']
 
-RuuviTagSensor.get_datas(handle_data, macs, run_flag)
+if __name_ == '__main__':
```

Comment:
Maybe for the sake of simplicity we can add `if __name_ == '__main__':` to the first example and leave it out from the rest? But it is good that it is shown and explained in the README.

Reply:
Oops, commented below instead of here yesterday; but yes I finally got around to doing that. The actual code examples still have the line. In the README it's only in the first example, with a paragraph saying it's required on Windows and macOS, but omitted in the rest of the examples in the README.